packages/fuels-test-helpers/src/signers.rs

```diff
@@ -6,15 +6,14 @@ use fuels_signers::LocalWallet;
 use std::net::SocketAddr;
 
 #[cfg(feature = "fuels-signers")]
-pub async fn setup_test_provider_and_wallet() -> (Provider, LocalWallet) {
+pub async fn setup_test_provider_and_wallet() -> LocalWallet {
```

Comment:
Add a docstring

Reply:
Was this actually resolved? I'm not seeing the docstring @vnepveu suggested.

Reply:
This was not updated actually

Reply:
Docstring added

Reply:
https://github.com/MujkicA/contract-deployment-qol/commit/55964a5f8eee1b6e7f88c3ac3727d755806a0166#diff-b655abf20ad2575234c19c2517d33a0a4e681de106149f287130a67f1f96155eR8